bats/tests/helpers/vm.bash

```diff
@@ -92,3 +76,13 @@ start_container_runtime() {
                --kubernetes-enabled=false 3>&-
     fi
 }
+
+start_application() {
+    "$RDCTL_resources" start \
+           --container-engine "$RD_CONTAINER_RUNTIME" \
+           --kubernetes-enabled \
+           --kubernetes-version "$RD_KUBERNETES_PREV_VERSION" \
+           --path-management-strategy rcfiles \
+           --kubernetes.suppress-sudo \
+           --updater=false
+}
```

Comment:
This function should extend `start_container_runtime`, which should drop the macOS specific code and use `rdctl start` everywhere. 

The options not specific to kubernetes (like `--updater` etc) should be moved to `start_application`.

The command should append `"$@"` as the last arguments, so `start_application` could be written like this:

```shell
start_kubernetes() {
  start_container_runtime \
    --kubernetes-enabled" \
    --kubernetes-version "$RD_KUBERNETES_PREV_VERSION"
}
```

We can do this in a followup PR, if your prefer.

Reply:
I am a bit confused as `start_container_runtime` runs open -a "Rancher Desktop" --args on MacOS. Does it do the same thing as `rdctl start`?

Reply:
Yes, I don't know why the code still uses `open -a ...`; it is maybe from before the time `rdctl start` was added. My suggestion is to get rid of it (needs to be tested), and then use `rdctl start` for all platforms.

Reply:
I've added a commit to your branch that shows what I'm talking about; please review and let me know if it looks good to you.